contrib/init/stacks.service

```diff
@@ -58,5 +59,9 @@ NoNewPrivileges=true
 # such as /dev/null, /dev/zero and /dev/random.
 PrivateDevices=true
 
+# Deny the creation of writable and executable memory mappings.
+MemoryDenyWriteExecute=true
```

Comment:
Since you're adding this, can we also add `SystemCallArchitectures=native`?
> [Specifically, it is recommended to combine this option with SystemCallArchitectures=native or similar](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#MemoryDenyWriteExecute=)

Reply:
hmm, i'll run some tests with this change but i'm not opposed to adding it.